api/lib/domain/usecases/get-ready-identity-providers.js

```diff
@@ -0,0 +1,5 @@
+const getReadyIdentityProviders = function ({ authenticationServiceRegistry }) {
+  return authenticationServiceRegistry.getReadyOidcProviderServices();
+};
+
+export { getReadyIdentityProviders };
```

Comment:
Si la modification est acceptée pour le nommage.

```suggestion
const getEnabledIdentityProviders = function ({ authenticationServiceRegistry }) {
  return authenticationServiceRegistry.getEnabledOidcProviderServices();
};

export { getEnabledIdentityProviders };
```